pinot-controller/src/main/java/com/linkedin/pinot/controller/validation/StorageQuotaChecker.java

```diff
@@ -63,12 +63,22 @@ public StorageQuotaChecker(AbstractTableConfig tableConfig, TableSizeReader tabl
   public QuotaCheckerResponse isSegmentStorageWithinQuota(@Nonnull File segmentFile, @Nonnull String tableNameWithType,
       @Nonnull String segmentName,
       @Nonnegative int timeoutMsec) {
-    Preconditions.checkNotNull(segmentFile);
-    Preconditions.checkNotNull(tableNameWithType);
-    Preconditions.checkNotNull(segmentName);
-    Preconditions.checkArgument(timeoutMsec > 0, "Timeout value must be > 0, input: %s", timeoutMsec);
-    Preconditions.checkArgument(segmentFile.exists(), "Segment file: %s does not exist", segmentFile);
-    Preconditions.checkArgument(segmentFile.isDirectory(), "Segment file: %s is not a directory", segmentFile);
+    try {
+      Preconditions.checkNotNull(segmentFile);
```

Comment:
Log info for user errors but this method is not right place to check that. Caller should handle bad input values before calling this method.

Preconditions explicitly say throw exception if that pre-requisite is not met. So, there should not be try-catch block around it. If you want to change semantics, chage those to if-statements